studio/storage_type.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
```

Comment:
Is this very useful or could testing attempted casting do this for us.

Reply:
It could for sure, but looking at simple enum value is cleaner in my opinion.